Seems there is some checkstyle error:
```
> Task :clients:checkstyleMain

[2022-06-16T18:57:40.716Z] [ant:checkstyle] [ERROR] /home/jenkins/jenkins-agent/workspace/Kafka_kafka-pr_PR-12304/clients/src/main/java/org/apache/kafka/clients/producer/internals/DefaultPartitioner.java:21:8: Unused import - org.apache.kafka.common.utils.Utils. [UnusedImports]
```

--- a/clients/src/main/java/org/apache/kafka/clients/producer/internals/DefaultPartitioner.java
+++ b/clients/src/main/java/org/apache/kafka/clients/producer/internals/DefaultPartitioner.java
@@ -18,7 +18,6 @@ package org.apache.kafka.clients.producer.internals;
 
 import org.apache.kafka.clients.producer.Partitioner;
 import org.apache.kafka.common.Cluster;
-import org.apache.kafka.common.utils.Utils;
 
 import java.util.Map;
 

